RFC/src/RFC-0010_CodeStructure.md

```diff
@@ -22,102 +22,101 @@ following conditions are met:
+The code follows a domain-driven design layout (DDD), with top-level directories falling into infrastructure, domain
 and application layers.
 
-### Infrastructure layer
+### Infrastructure Layer
 
-This layer provides a set of crates which have general infrastructural utility. The rest of the Tari codebase can make use
+The infrastructure layer provides a set of crates that have general infrastructural utility. The rest of the Tari codebase can make use
 of these crates to obtain persistence, communication and cryptographic services. The infrastructure layer doesn't know
-anything about blockchains, transactions, or digital assets.
+anything about blockchains, transactions or digital assets.
 
-We recommend that code in this layer generalises infrastructure services behind abstraction layers as much as is
+We recommend that code in this layer generalizes infrastructure services behind abstraction layers as much as is
 reasonable, so that specific implementations can be swapped out with relative ease.
 
-### Domain layer
+### Domain Layer
 
-The Domain layer houses the Tari "business logic". All protocol-related concepts and procedures are defined and
+The domain layer houses the Tari "business logic". All protocol-related concepts and procedures are defined and
 implemented here.
 
-This entails that any and all terms defined in the [Glossary] will have a software implementation here, and only here.
-They can be _used_ in the Application layer, but must be implemented in the Domain layer.
+This means that any and all terms defined in the [Glossary] will have a software implementation here, and only here.
+They can be _used_ in the application layer, but must be *implemented* in the domain layer.
 
 The domain layer can make use of crates in the infrastructure layer to achieve its goals.
 
-### Application layer
+### Application Layer
 
-Applications build on top of the domain layer to produce the executable software that is deployed as part of the Tari
-network.
+In the application layer, applications build on top of the domain layer to produce the executable software that is 
+deployed as part of the Tari network.
 
 As an example, the following base layer applications may be developed as part of the Tari protocol release:
 
 * A standalone miner (tari_miner)
 * A pool miner (tari_pool_miner)
-* A CLI wallet for the Tari cryptocurrency (cli_wallet)
+* A command line interface (CLI) wallet for the Tari cryptocurrency (cli_wallet)
 * A base node executable (tari_basenode)
-* A REST API server for the base node
+* A REST Application Programming Interface (API) server for the base node
 
-### Code layout
+### Code Layout
 
-The top-level source code directories in this repository reflect the respective [DDD] layers; except that there are two
+The top-level source code directories in this repository reflect the respective (DDD) layers; except that there are two
```

Comment:
This `(DDD)` must change back to `[DDD]` otherwise the link to Wiki Pedia does not work.